### ISSUE TITLE
Technical report or accompanying Paper

### ISSUE THREAD

**Issue by miangoleh** [Stable-X/StableDelight#3]
Hi!

I was wondering if there is a technical report or an accompanying paper that describes the underlying algorithm in more detail.

**Comment by hugoycj** [comment]
Hi Miangoleh!Thank you for your interests! StableDelight is just an open-source project and we don't have plan to write a technical report yet. It builds upon the first-stage YOSO from [StableNormal](https://github.com/Stable-X/StableNormal), with different training data and additional loss function. The training data include Hypersim, Lumos, and various Specular Highlight Removal datasets from TSHRNet. Also, we've integrated a multi-scale SSIM loss into our diffusion training process, to improve sharpness in one-step diffusion prediction.

We're currently in the optimization phase and would greatly value any feedback!
